docs/plugins/ContentIndex.md

```diff
@@ -0,0 +1,24 @@
+---
+title: ContentIndex
+tags:
+  - plugin/emitter
+---
+
+This plugin creates a comprehensive index of the site's content, generating additional resources like a sitemap and RSS feed.
```

Comment:
```suggestion
This plugin emits RSS as well as a XML site map for your site. Usually, this is useful for general net neutrality.
```

Reply:
It also creates a `contentIndex.json` file which is used by dynamic frontend components like search and graph